.github/workflows/build_and_release_cuda.yml

```
@@ -198,8 +209,9 @@ jobs:
         with:
           cuda-version: "12.4.1"
           cuda-compute-capability: ${{ matrix.target.compute_cap }}
-          artifact-tag: ${{ matrix.target.target_os }}_${{ matrix.target.target_arch }}
+          artifact-tag: ${{ matrix.target.target_os }}_${{ matrix.target.target_arch }}_${{ env.RUST_PROFILE }}
```

```suggestion
          artifact-tag: ${{ matrix.target.target_os }}_${{ matrix.target.target_arch }}
```

I don't think we need to tag the artifact with the profile